doc/source/intro.rst

```diff
@@ -1,3 +1,6 @@
+N-dimensional labelled arrays
```

Comment:
Wouldn't it be simpler to include the corresponding chunks of the README directly in index.rst instead of doing it via intro.rst?